### PR TITLE
[CU-86b2hmmmv] Add cli commands to interact with workflow defaults 

### PR DESCRIPTION
**Pull request by patmagee**

### CODE REVIEW

**Comment by platform-automation-dnastack** [comment]
Task linked: [CU-86b2ek702 Add CLI commands to interact with Workflow Defaults](https://app.clickup.com/t/86b2ek702)
Task linked: [CU-86b2hmmmv merge instrument integration](https://app.clickup.com/t/86b2hmmmv)

File: dnastack/alpha/cli/workbench/workflows_commands.py [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+             ArgumentSpec(
+                 name='values',
+                 arg_names=['--values'],
+                 help='The values to use when creating the defaults',
+                 as_option=True
+             ),
+             ArgumentSpec(
+                 name='name',
+                 arg_names=['--name'],
+                 help='The values to use when creating the defaults',
```
Comment (mpanik) [inline dnastack/alpha/cli/workbench/workflows_commands.py:87]: copy pasta error

File: dnastack/alpha/cli/workbench/workflows_commands.py [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
```
Comment (mpanik) [inline dnastack/alpha/cli/workbench/workflows_commands.py:284]: same

**Review by mpanik — APPROVED** [review]
LGTM!

**Pull request merged**